docs/commandline/start.md

```diff
@@ -0,0 +1,40 @@
+# start
+
+start command is used to invoke a container with the container name or ID.
+
+## Description
+
+The `pouch start` command starts a container in `created` or `exited` state and runs the user-specified program. `container name` or `container ID` must be specified, if the container doesn't exist this command will return error.
+
+## Help Information
+Use the following command to get the usage of "pouch start":
+
+```
+# pouch start -h
+# pouch start --help
+```
+
+## Flag Guidance
+
+### -a, --attach        
+
+Attach the console to cotainer's STDOUT and STDERR. 
+
+###   -i, --interactive
```

Comment:
More white spaces there before `-i`. And there should be exactly one white space.